src/components/Recipes/RecipeItem.css

```diff
@@ -64,4 +64,14 @@ background-color: #468ccf;
 .button:active {
 position: relative;
 top: 1px;
-} 
\ No newline at end of file
+} 
+
+.missingIngredients > p {
+  font-size: 15px;
+  text-decoration-line: underline;
+}
+
+.usedIngredients > p {
+  font-size: 15px;
```

Comment:
we will have to refactor px to rem (px is static aka never changes, rem changes in regards to the parent's size)

Reply:
rem vs em vs px is a good learn

Reply:
Got it. Will change.